### DISCUSSION TITLE
Alternative SummerWind image which does not require root permissions

### DISCUSSION THREAD

**Discussion by ajitkumarnayak1976** [actions/actions-runner-controller#1117] [General]
The current SummerWind image used for this actions runner controller leverages "Docker Dind" (docker in docker) for dependency management. This is a concern in our organization, our security department would prefer leveraging an image which uses generic permissions without system-wide access if possible. Are we able to get an alternative version of the image for this summerwind image which does not leverage docker dind?

https://github.com/actions-runner-controller/actions-runner-controller/blob/master/charts/actions-runner-controller/values.yaml#L57

**Comment by mumoshu** [comment]
Try sysbox https://github.com/actions-runner-controller/actions-runner-controller/discussions/977 :)

**Reply by shichengripple001** [reply]
or is it possible to support kaniko?

**Reply by ajitkumarnayak1976** [reply]
we are using EKS optimized OS Images from AWS on Amazon Linux 2, is there support planned for that with this sysbox?
